app/models/bookmarked-content.js

```diff
@@ -10,10 +10,6 @@ export default Model.extend({
   route: Ember.computed('contest', 'problem', 'contestTypeId', function() {
     const contest_type = this.contest.get('contest_type')
     switch (contest_type) {
-      case 'practice': return {
```

Comment:
we need this 
<img width="1300" alt="Screenshot 2021-05-29 at 10 09 50 PM" src="https://user-images.githubusercontent.com/18213029/120077996-dab2a680-c0ca-11eb-8e2e-22d50108fd95.png">
instead of this
<img width="1284" alt="Screenshot 2021-05-29 at 10 09 40 PM" src="https://user-images.githubusercontent.com/18213029/120078002-e30ae180-c0ca-11eb-887c-74b4e877dc4d.png">